tpc/pregen/media.dsl/dslplatform/Guards.java

```diff
@@ -0,0 +1,15 @@
+package serializers.dslplatform;
+
+public final class Guards {
+    /// No-oped for perf reasons
```

Comment:
Was this the exact generated code or was this altered by hand?

I realize that it's sort of unfair that some serializers do less checking, but I think it's more important to show how fast something is in the typical configuration.  For example Protobuf makes a copy of the data it's given so it can guarantee immutability.  It takes a performance hit for this, but that's the performance a typical user will expect to see.

It would be nice to document this in the results though.  For example, we track certain properties of serializers (e.g. "tree" vs "graph").  We could maybe also track properties of object creation -- "null-checked", "immutable", etc.